lisa/tools/stress_ng.py

```diff
@@ -47,6 +48,20 @@ def launch(
             cmd += f" --timeout {timeout_in_seconds} "
         self.run(cmd, force_run=True)
 
+    def launch_job_async(self, job_file: str, sudo: bool = False) -> Process:
+        return self.run_async(f"--job {job_file}", force_run=True, sudo=sudo)
+
+    def launch_class_async(
```

Comment:
Can you change the default `launch` method to be more accurate name? It looks there shouldn't have a default `launch` method, because the stress_ng handles different scenarios.

Reply:
Sure, renamed it to a proper name.